HW_5_2023/task_34.py

```diff
@@ -0,0 +1,22 @@
+'''Задача 34:Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+Поскольку разобраться в его кричалках не настолько просто, насколько 
+легко он их придумывает, Вам стоит написать программу. Винни-Пух считает, 
+что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе 
+стихотворения одинаковое. Фраза может состоять из одного слова, 
+если во фразе несколько слов, то они разделяются дефисами. 
+Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу 
+с клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и 
+“Пам парам”, если с ритмом все не в порядке
+*Пример:*
+**Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+    **Вывод:** Парам пам-пам'''
+poem = input("Введите стихотворение Винни-Пуха: ")
+phrases = poem.split()
+text = []
+for phrase in phrases:
+    text_count = sum([1 for letter in phrase if letter in "аеёиоуыэюя"])
```

Comment:
Можно проще 
```suggestion
    text_count = sum(letter in "аеёиоуыэюя" for letter in phrase)
```
Подумайте, почему так тоже будет работать